x-pack/plugins/infra/public/pages/metrics/layouts/host.ts

```diff
@@ -16,37 +17,54 @@ import {
 export const hostLayoutCreator: InfraMetricLayoutCreator = theme => [
   {
     id: 'hostOverview',
-    label: 'Host',
+    label: i18n.translate('xpack.infra.hostMetrics.hostLabel', {
```

Comment:
Matching the container scheme above, this could become `xpack.infra.metricDetailPage.hostMetricsLayout.layoutLabel`?